### PR TITLE
v7.12.1

### PR DESCRIPTION
**Pull request by romgrk**

### CODE REVIEW

**Review by alexfauquette — APPROVED** [review]

**Review by noraleonte — APPROVED** [review]

File: packages/x-charts-pro/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-alpha.0",
+  "version": "7.0.0",
```
Comment (JCQuintas) [inline packages/x-charts-pro/package.json:3]: @alexfauquette is this correct?

Comment (LukasTy) [inline packages/x-charts-pro/package.json:3]: The changelog says that it is not and we also agreed during the meeting that the pre-release phase is expected and decided upon. 👌
```suggestion
  "version": "7.0.0-alpha.0",
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,74 @@
...
+Same changes as in `@mui/x-date-pickers@7.12.1`.
+
+### Charts
+
+#### `@mui/x-charts@7.12.1`
+
+- [charts] Fix charts vendor publish config (#14073) @JCQuintas
+- [charts] Move `plugins` to `PluginProvider` (#14056) @JCQuintas
+
+#### `@mui/x-date-charts-pro@__VERSION-ALPHA__` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
```
[the remark this answers is not in the document]
Comment (alexfauquette) [reply CHANGELOG.md:54]:
```suggestion
#### `@mui/x-date-charts-pro@7.0.0-alpha.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
```

File: packages/x-charts-pro/package.json (hunk 1 above)
Comment (alexfauquette) [reply packages/x-charts-pro/package.json]: Effectively, I missed this error

**Review by MBilalShafi — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,74 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.1
+
+_Aug 8, 2024_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎨 Charts get a new component to display color mapping in the legend.
```
Comment (cherniavskii) [inline CHANGELOG.md:12]: Nitpick
```suggestion
- 🎨 Charts get a new component to display color mapping in the legend
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,74 @@
...
+- 🚀 The `@mui/x-charts-pro` is released in alpha version 🧪. This new package introduces two main features:
+  - The Heatmap component
+  - The zoom interaction on the bar, line, and scatter charts
```
Comment (cherniavskii) [inline CHANGELOG.md:15]:
```suggestion
  - The [Heatmap component](https://mui.com/x/react-charts/heatmap/)
  - The [zoom interaction](https://mui.com/x/react-charts/zoom-and-pan/) on the bar, line, and scatter charts
```

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (cherniavskii) [reply CHANGELOG.md:15]: Also, should it be a minor version bump since we release new features for Charts?
cc @alexfauquette

Comment (LukasTy) [reply CHANGELOG.md:15]: Those features are available only on the `-pro` package, which is being released only now and as `alpha.0`.
Not sure if that warrants a minor bump. 🤔

Comment (romgrk) [reply CHANGELOG.md:15]: Afaict pre-releases fall under the same category as `0.x` releases, aka "do whatever you want", in other words they fall outside of semver guarantees. That's how I understand the spec: https://semver.org/#spec-item-9

**Pull request merged**